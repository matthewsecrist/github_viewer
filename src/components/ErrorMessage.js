import React from 'react'
import { string } from 'prop-types'
import { Notification } from 'rbx'

const ErrorMessage = ({ message }) => (
  <Notification color='danger'>{message}</Notification>
)

ErrorMessage.propTypes = {
  message: string
}

export default ErrorMessage
