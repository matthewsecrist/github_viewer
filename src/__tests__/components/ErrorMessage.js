import React from 'react'
import { render, cleanup } from '@testing-library/react'
import ErrorMessage from '../../components/ErrorMessage'

afterEach(cleanup)

describe('<ErrorMessage />', () => {
  it('matches the snapshot', () => {
    const { container } = render(<ErrorMessage message='mock-message' />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
