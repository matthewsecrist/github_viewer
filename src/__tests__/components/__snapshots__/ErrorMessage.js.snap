// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorMessage /> matches the snapshot 1`] = `
<div
  class="notification is-danger"
>
  mock-message
</div>
`;
